pysigma/signatures.py

```diff
@@ -91,35 +91,36 @@ def sigma_string_to_regex(original_value: str):
 
     return re.compile(''.join(full_content), flags=re.IGNORECASE)  # Sigma strings are case insensitive
```

Comment:
It might be worth returning only the regex string here without compiling it. Since we recompile it later anyway. If we end up importing a large number of signatures it could impact starting time.